server/config/conditions.ts

```diff
@@ -194,12 +194,14 @@ export default {
       code: '4858cd8b-bca6-4f11-b6ee-439e27216d7d',
       category: 'Making or maintaining contact with a person',
       text: 'Not to seek to approach or communicate with [INSERT NAME OF VICTIM AND / OR FAMILY MEMBERS] without the prior approval of your supervising officer and / or [INSERT NAME OF APPROPRIATE SOCIAL SERVICES DEPARTMENT].',
+      tpl: 'Not to seek to approach or communicate with {name} without the prior approval of your supervising officer and / or {socialServicesDepartment}.',
```

Comment:
So here, social services department is optional... I guess thats in the pipeline

Reply:
Yes, this one and the one where we have an Other review period are not done yet.